src/Microsoft.CodeAnalysis.Analyzers/Core/ImplementationIsObsoleteAnalyzer.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the MIT license.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Immutable;
+using System.Linq;
+using Analyzer.Utilities;
+using Analyzer.Utilities.Extensions;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Microsoft.CodeAnalysis.Analyzers
+{
+    using static CodeAnalysisDiagnosticsResources;
+
+    [DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)]
+    public class ImplementationIsObsoleteAnalyzer : DiagnosticAnalyzer
+    {
+        private const string ImplementationIsObsoleteAttributeName = "ImplementationIsObsoleteAttribute";
```

Comment:
📝 This name does not match dotnet/roslyn#75223

Reply:
Fixed the roslyn side PR.